topics/lua-api.md

```diff
@@ -411,21 +405,32 @@ The named argument version except the following named arguments:
 * _flags_ - Lua array of strings representing function flags
 * _description_ - Lua string representing the function description
 
-_function\_name_ and _callback_ are mandatory, the other named arguments are option. Error raise on the following cases:
-
-* Missing mandatory arguments
-* Unkown named argument
-* Wrong argument type
-* Unknows flag
-* Another function with the same name already registered on the current library
+_function\_name_ and _callback_ are mandatory, the other named arguments are option.
 
 Usage Example:
 
 ```
 redis> FUNCTION LOAD Lua mylib "redis.register_function{function_name='noop', callback=function() end, function, flags={'no-writes'}, description='some desc')"
 ```
 
-For more information about function flags refer to [Function Flags](functions-intro#function-flags)
+#### <a name="script_flags"></a> Script Flags
+
+The following is a list of supported script flags:
+
+* `no-writes` - indicating the function perform no writes which means that it is OK to run it on:
+   * read-only replica
+   * Using FCALL_RO
+   * If disc error detected
+   
+   It will not be possible to run a function in those situations unless the function turns on the `no-writes` flag, in this case Redis will inforce the function is doing not writes and will raise an error if it does.
```

Comment:
```suggestion
   It will not be possible to run a function in those situations unless the script turns on the `no-writes` flag, in this case Redis will enforce the script is doing not writes and will raise an error if it does.
```